docs/Update-SSHKeys.md

```diff
@@ -49,6 +49,10 @@ spec:
  - Updating `user: name`: Do not update the `user: name` field. The only user currently supported is `core` as shown in the above example config.
 
  ## Applying the MachineConfig
+First scale the Cluster Version Operator:
+```sh
+    oc -n openshift-cluster-version scale --replicas=0 deployment/cluster-version-operator
```

Comment:
Is this for testing purposes only? In actual cluster operation we would need the CVO still for updates, etc. no?

Reply:
Ah good call.  I'll leave it out!

Thanks for the quick reply, @yuqi-zhang !